### PR TITLE
Create 2023-05-09-storage.md

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-dprice-05-09-storage-release-notes-neondatabase.vercel.app/docs/release-notes

Note: Added in a completely unrelated formatting fix for another file.

### CODE REVIEW

**Comment by danieltprice** [comment]
Question: What is the standby mode issue that was fixed? When do we use it & what do we use it for? I'd like to add some context to the release note.

File: content/release-notes/2023-05-09-storage.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Docker image files are now included in the [neon](https://github.com/neondatabase/neon) GitHub repository.
```
Comment (vadim2404) [inline content/release-notes/2023-05-09-storage.md:7]: they were here, but, I hope, that now it's possible to use it everyone (we make docker images public)

File: content/release-notes/2023-05-09-storage.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- Safekeepers: Enabled parallel offload of WAL segments to remote storage. This change allows Safekeepers to upload up to five segments concurrently.
+
+### Bug fixes
+
+- Compute: Addressed an issue that prevented PostgreSQL 15 from entering standby mode.
```
Comment (vadim2404) [inline content/release-notes/2023-05-09-storage.md:12]: It's related to replication. However, I'm not sure that it's the right explanation.

@lubennikovaav, @MMeent, can you help?

Comment (danieltprice) [reply content/release-notes/2023-05-09-storage.md:12]: Thank you. Let's remove it until we release the feature.

File: content/release-notes/2023-05-09-storage.md (hunk 1 above)
Comment (NanoBjorn) [reply content/release-notes/2023-05-09-storage.md:7]: Docker images already were in public Docker Hub and Dockerfiles were also available, but at the same time we are building and pushing them into private registry (ECR). By default, there was private registry used in `FROM ...` clauses in Dockerfile making it impossible for people from outside to run `docker build . -t neondatabase/neon` without knowledge of what should be changed to make it work. I changed defaults to public registry so now `docker build .` will work without any extra access

Comment (danieltprice) [reply content/release-notes/2023-05-09-storage.md]: Thank you. Can you take a look at the latest revision?

**Review by vadim2404 — APPROVED** [review]

**Review by vadim2404 — APPROVED** [review]

**Pull request merged**